pulsar-package-management/bookkeeper-storage/src/main/java/org/apache/pulsar/packages/management/storage/bookkeeper/BookKeeperPackagesStorageConfiguration.java

```diff
@@ -35,15 +35,15 @@
     }
 
     int getNumReplicas() {
```

Comment:
what about naming this method` getPackagesReplicas() `?

Reply:
Thank you. I will fix it.